cylc/flow/parsec/jinja2support.py

```diff
@@ -137,7 +137,7 @@ def _load_jinja2_extensions():
     return jinja2_extensions
 
 
-def jinja2environment(dir_=None):
+def jinja2environment(dir_=None, template_vars=None):
```

Comment:
```suggestion
def jinja2environment(dir_=None):
```

Reply:
@wxtim, looks like you've blasted this away with a force push.

Reply:
Is this resolved?